src/scss/alert/_mixins.scss

```diff
@@ -1,4 +1,4 @@
-@mixin oMessageAlertIcon ($icon-name, $icon-color, $icon-size) {
```

Comment:
Could we please leave the mixins as an `oMessageAlert...`? 

We may want to cater for other types of messages further down the line (such as cookie messages/ marketing banners) and it would be useful to have the distinction in the names of the mixins. 

The inline  message is still an 'alert' type message, its just presented in a different format.